README.md

```diff
@@ -657,17 +657,24 @@ the users understand the source explicitly, although this is not strictly requir
 
 | Artifact Type      | Description                                                                          |
 |--------------------|--------------------------------------------------------------------------------------|
-| `torch.save`       | A model artifact obtained by [Serialized Pickle Object][pytorch-save] (i.e.: `.pt`). |
-| `torch.jit.script` | A model artifact obtained by [`TorchScript`][pytorch-jit-script].                    |
-| `torch.export`     | A model artifact obtained by [`torch.export`][pytorch-export] (i.e.: `.pt2`).        |
-| `torch.compile`    | A model artifact obtained by [`torch.compile`][pytorch-compile].                     |
+| `torch.save`       | A [serialized python pickle object][pytorch-save] (i.e.: `.pt`) which can represent a model or state_dict. |
+| `torch.jit.save` | A [`TorchScript`][pytorch-jit-script] model artifact obtained with one or more of the graph export options Torchscript Tracing and Torchscript Scripting.                    |
+| `torch.export.save`     | A model artifact storing an ExportedProgram obtained by [`torch.export.export`][pytorch-export] (i.e.: `.pt2`).        |
+|  `TFSavedModel`   | A [SavedModel][tf-save] from Tensorflow or Keras. |
```

Comment:
Shouldn't we use something like `tf.saved_model.save` pointing at either https://www.tensorflow.org/guide/saved_model#saving_a_custom_model or https://www.tensorflow.org/api_docs/python/tf/saved_model/save to keep the naming convention consistent with a reference function?

Reply:
yeah I was going back and forth on this. Keras save functions have different argument options to export in different formats. I'm not sure yet if using the function name was the best mapping to artifact type for keras and TF. the actual function exporting a specific artifact type might not be in the public api, need to check docs more thoroughly.

Reply:
switched to use function names